legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/testDataGeneration/testDataGeneration.pure

```diff
@@ -847,7 +847,7 @@ function meta::relational::testDataGeneration::executionPlan::planTestDataGenera
          |  let nestedNodes = meta::relational::testDataGeneration::executionPlan::planTestDataGenerationForNestedViewTree($tree, $runtime, $exeCtx, $dbConnection, $relationColumnMap, $currentIndices, true, [], $config, $extensions);
             if($nestedNodes->exists(x | $x->instanceOf(ErrorExecutionNode)),
                | $nestedNodes,
-               | let viewResultVarName = 'res_' + $currentIndices->map(x | $x->toString())->joinStrings('_');
+               | let viewResultVarName = $nestedNodes->at(0)->cast(@AllocationExecutionNode).varName;
```

Comment:
Does this always work? Becz, post processors can add more nodes (like query tags in SF)

Reply:
Those are wrapped inside allocation node.
Ref : https://github.com/finos/legend-engine/blob/master/legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/testDataGeneration/testDataGeneration.pure#L1122